bindings/python/examples/client/stronghold.py

```diff
@@ -1,6 +1,7 @@
 from iota_sdk import Client, StrongholdSecretManager, SecretManager
 from dotenv import load_dotenv
 import os
+import sys
```

Comment:
Just curious, why is sys needed?

Reply:
To exit if there is a missing env var, the print can probably replaced by some exception to have both at once
```
if 'NON_SECURE_USE_OF_DEVELOPMENT_MNEMONIC_1' not in os.environ:
    print(".env mnemonic is undefined, see .env.example")
    sys.exit(1)
```

Reply:
Ah yeah, I hate interpreted languages